app/services/workers/create-workload.js

```diff
@@ -85,18 +85,18 @@ module.exports.execute = function (task) {
       var taskDetails = {
         workloadBatch: new Batch(insertedWorkloadIds[0], insertedWorkloadIds.length)
       }
-      var calculateWorkloadPointsTask = new Task(
+      var reductionsWorkerTask = new Task(
                 undefined,
                 submittingAgent.WORKER,
-                taskType.CALCULATE_WORKLOAD_POINTS,
+                taskType.PROCESS_REDUCTIONS,
                 taskDetails,
                 workloadReportId,
                 undefined,
                 undefined,
                 taskStatus.PENDING
                 )
 
-      return createNewTasks([calculateWorkloadPointsTask])
+      return createNewTasks([reductionsWorkerTask])
             .then(function () {
               logger.info('Tasks created')
```

Comment:
Add task type to logger info as in file below

Reply:
added